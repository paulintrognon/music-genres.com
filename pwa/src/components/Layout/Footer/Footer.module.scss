.container {
  width: 100%;
  height: 3rem;
  line-height: 3rem;
  text-align: center;
  font-size: var(--font-size-13);
  color: #FFF;
}

.content {
  max-width: 40rem;
  border-top: 1px #fff solid;
  margin: auto;
  display: flex;
  align-items: center;
  justify-content: center;
}

.plus {
  padding: 0 0.5rem;
}