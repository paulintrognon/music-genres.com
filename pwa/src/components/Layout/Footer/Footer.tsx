import PlusIcon from '../../Shared/Atoms/Icons/PlusIcon/PlusIcon'
import classes from './Footer.module.scss'

const Footer: React.FunctionComponent = () => {
  return (
    <footer className={classes.container}>
      <div className={classes.content}>
        <p>
          <a href="https://fr.linkedin.com/in/margotbrun" target="_blank" rel="noreferrer">
            Design by Margot Brun
          </a>
        </p>
        <PlusIcon className={classes.plus} alt="-" />
        <p>
          <a
            href="https://github.com/paulintrognon/music-genres.com"
            target="_blank"
            rel="noreferrer"
          >
            Code
          </a>{' '}
          by{' '}
          <a href="https://paulintrognon.fr/" target="_blank" rel="noreferrer">
            Paulin Trognon
          </a>
        </p>
      </div>
    </footer>
  )
}
export default Footer
