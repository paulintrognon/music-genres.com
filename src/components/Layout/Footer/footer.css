.main-footer {
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 50px;
  line-height: 50px;
  text-align: center;
  font-size: 0.9rem;
  color: #FFF;
  border-top: 1px #fff solid;
}
