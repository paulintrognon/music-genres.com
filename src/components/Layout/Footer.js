import React from 'react';

import './footer.css';

export default function () {
  return (
    <footer className="main-footer">
      <a href="https://github.com/paulintrognon/" target="_blank" rel="noopener noreferrer">
        Paulin Trognon
      </a>
      &nbsp;-&nbsp;
      <a href="https://github.com/paulintrognon/music-genres.com" target="_blank" rel="noopener noreferrer">
        <i className="fa fa-github" aria-hidden="true"></i>
        &nbsp;Code source
      </a>
    </footer>
  )
}
