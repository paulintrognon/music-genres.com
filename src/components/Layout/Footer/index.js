import React from 'react';

import './footer.css';

import plusImage from '../../../images/plus.png';

export default class Footer extends React.Component {
  render() {
    return (
      <footer className="main-footer">
        <div className="main-footer-content">
          Design by&nbsp;
          <a href="https://fr.linkedin.com/in/margotbrun" target="_blank" rel="noopener noreferrer">
            Margot Brun
          </a>
          &nbsp;<img src={plusImage} className="plus-sign" alt="-" />&nbsp;
          <a href="https://github.com/paulintrognon/music-genres.com" target="_blank" rel="noopener noreferrer">
            Code by&nbsp;
          </a>
          <a href="https://paulintrognon.fr/" target="_blank" rel="noopener noreferrer">
            Paulin Trognon
          </a>
        </div>
      </footer>
    );
  }
}
