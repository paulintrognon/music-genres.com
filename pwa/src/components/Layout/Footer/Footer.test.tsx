import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'

it('should render the Footer', () => {
  expect(renderer.create(<Footer />).toJSON()).toMatchSnapshot()
})
