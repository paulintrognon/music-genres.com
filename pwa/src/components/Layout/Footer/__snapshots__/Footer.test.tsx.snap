// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Footer 1`] = `
<footer
  className="container"
>
  <div
    className="content"
  >
    <p>
      <a
        href="https://fr.linkedin.com/in/margotbrun"
        rel="noreferrer"
        target="_blank"
      >
        Design by Margot Brun
      </a>
    </p>
    <img
      alt="-"
      className="plus"
      src="plus.png"
    />
    <p>
      <a
        href="https://github.com/paulintrognon/music-genres.com"
        rel="noreferrer"
        target="_blank"
      >
        Code
      </a>
       
      by
       
      <a
        href="https://paulintrognon.fr/"
        rel="noreferrer"
        target="_blank"
      >
        Paulin Trognon
      </a>
    </p>
  </div>
</footer>
`;
